libsweep/examples/README.md

```diff
@@ -2,12 +2,13 @@
 
 Examples for the `libsweep` library.
 
-Requires `libsweep.so` to be installed.
 This can be either the dummy library always returning static point cloud data or the device library requiring the Scanse Sweep device to be plugged in.
 
 ### Quick Start
 
-To build: 
+#### To build on Linux: 
+
+Requires `libsweep.so` be installed, in `"/usr/lib"` for example.
```

Comment:
I would remove the `/usr/lib` part. If not it would be more helpful for the user to know how to change linker and loader search paths.